Jellyfin.Plugin.KodiSyncQueue/Jellyfin.Plugin.KodiSyncQueue.csproj

```diff
@@ -15,7 +17,22 @@
     <PackageReference Include="Jellyfin.Data" Version="10.*-*" />
     <PackageReference Include="Jellyfin.Controller" Version="10.*-*" />
     <PackageReference Include="LiteDB" Version="4.1.4" />
-    <PackageReference Include="Microsoft.AspNetCore.Mvc" Version="2.2.0" />
   </ItemGroup>
 
+  <!-- Code Analyzers-->
+  <ItemGroup>
```

Comment:
We generally only include these for debug builds.
Not sure how these behave when publishing a release

Reply:
I pulled that part from musicbrainz plugin.
I remember that this was done intentionally at the time, but I don't remember in which repo this was discussed or the reasoning behind it.

I do however generally build release builds, even when testing, and I'm more likely to see the errors from the analyzers this way.

(I'm not saying either way is the right way.)